cli/src/init.rs

```diff
@@ -283,18 +286,48 @@ pub fn generate_grammar_files(
     }
 
     // Write .gitignore file
-    missing_path(repo_path.join(".gitignore"), |path| {
-        generate_file(path, GITIGNORE_TEMPLATE, language_name, &generate_opts)
-    })?;
+    missing_path_else(
+        repo_path.join(".gitignore"),
+        allow_update,
+        |path| generate_file(path, GITIGNORE_TEMPLATE, language_name, &generate_opts),
+        |path| {
+            let mut contents = fs::read_to_string(path)?;
+            if !contents.contains("Zig artifacts") {
+                contents = contents.replace(
+                    "*.lib",
+                    indoc! {"
+                    *.lib
+
+                    # Zig artifacts
+                    .zig-cache/
+                    zig-cache/
+                    zig-out/
```

Comment:
Every grammar doesn't have *.lib, as this was added very recently, so this replacement does nothing.